src/main/java/uk/gov/hmcts/darts/event/controller/EventsController.java

```diff
@@ -50,9 +50,8 @@ public class EventsController implements EventApi {
     public ResponseEntity<AddDocumentResponse> eventsPost(
         @Parameter(name = "DartsEvent", description = "") @Valid @RequestBody(required = false) DartsEvent dartsEvent
     ) {
-
         // send to darts-gateway for DAR PC Notify
```

Comment:
Please could you remove this redundant comment for me - since you have covered the complexities of when to handle the darNotify now. Thanks